sdk/eventhubs/azure_messaging_eventhubs/README.md

```diff
@@ -157,80 +157,127 @@ More information about Event Hubs features and terminology can be found here: [l
 Examples for various scenarios can be found on in the samples directory in our GitHub repo for
 [Event Hubs](https://github.com/Azure/azure-sdk-for-rust/tree/main/sdk/eventhubs/azure-messaging-eventhubs/samples).
 
-## Send events
-
-The following example shows how to send events to an event hub:
+## Open an eventhubs message producer on an eventhubs instance.
 
 ```rust
-async fn send_events() {
-    let host = std::env::var("EVENTHUBS_HOST").unwrap();
-    let eventhub = std::env::var("EVENTHUB_NAME").unwrap();
+use azure_messaging_eventhubs::{ProducerClient, ProducerClientOptions};
 
-    let credential = azure_identity::DefaultAzureCredential::new().unwrap();
+async fn open_producer_client() -> Result<ProducerClient, Box<dyn std::error::Error>>
```

Comment:
Would we always return an `azure_core::Error`? If so, just be specific:
```suggestion
async fn open_producer_client() -> azure_core::Result<ProducerClient>
```
That way the compiler can use static dispatch.

Reply:
I had the generic form because std::env returned a `VarError`. But I don't need the std::env reference.